mm/page_alloc.c

```diff
@@ -3547,16 +3627,29 @@ static unsigned long nr_free_zone_pages(int offset)
 	return sum;
 }
 
+
+
+
 /**
  * nr_free_buffer_pages - count number of pages beyond high watermark
  *
  * nr_free_buffer_pages() counts the number of pages which are beyond the high
  * watermark within ZONE_DMA and ZONE_NORMAL.
  */
+#ifndef CONFIG_LIB
```

Comment:
is it possible to change deeper part (nr_free_zone_pages) ?
note that if possible I would like to minimize the modification on other part of kernel code (again if possible).